Audio.ios.js

```diff
@@ -11,11 +11,28 @@ var AudioPlayerManager = NativeModules.AudioPlayerManager;
 var AudioRecorderManager = NativeModules.AudioRecorderManager;
 
 var AudioPlayer = {
-  play: function(path) {
-    AudioPlayerManager.play(path);
+  play: function(path, options) {
+    var playbackOptions = null;
+
+    if (!options) {
+      playbackOptions = {
+        sessionCategory: 'SoloAmbient'
+      };
+    } else {
+      playbackOptions = options;
+    }
+    AudioPlayerManager.play(path, playbackOptions);
```

Comment:
Glad you made this PR, the functionality is great to have! You could simplify the content of these methods somewhat using the OR operator.. perhaps something like:

```javascript
play: function(path, options) {
  options = options || {sessionCategory: 'SoloAmbient'};
  AudioPlayerManager.play(path, options);
}
```